api/client/commands.go

```diff
@@ -1929,6 +1929,10 @@ func (cli *DockerCli) CmdLogs(args ...string) error {
 		return err
 	}
 
+	if env.GetSubEnv("HostConfig").GetSubEnv("LogConfig").Get("Type") != "json-file" {
```

Comment:
@LK4D4 hmm it's happening because daemon doesn't have your changes yet. we use another Docker daemon we periodically update. @jfrazelle